srcpkgs/gstreamer1/template

```diff
@@ -1,45 +1,34 @@
 # Template file for 'gstreamer1'
 pkgname=gstreamer1
-version=1.16.2
+version=1.18.1
 revision=1
 wrksrc="gstreamer-${version}"
 build_style=meson
 build_helper="gir"
 configure_args="-Dptp-helper-permissions=capabilities
  -Dpackage-origin=https://www.voidlinux.org -Ddbghelp=disabled
- -Dintrospection=$(vopt_if gir enabled disabled)
- -Dgtk_doc=$(vopt_if gtk_doc enabled disabled)"
+ -Dintrospection=$(vopt_if gir enabled disabled)"
 hostmakedepends="gettext pkg-config flex python docbook-xsl glib-devel
- libcap-progs $(vopt_if gtk_doc gtk-doc)"
```

Comment:
Why was this rmeoved?

Reply:
Doesn't actually build even if enabled, and option was correct, so it is something with upstream

Reply:
Why remove it, then? Just remove it as default for now, ask upstream to fix. And definitely leave a comment in the template and in the commit message about it.

Reply:
it's probably fine to remove, especially considering gtkdoc is fucky with crosstargets; i doubt that anybody will read local gtkdoc html documentation either way